internal/policies/client.go

```diff
@@ -90,23 +106,25 @@ func (f *Client) GetClusterWidePolicies(ctx context.Context) (*Policies, error)
 
 	clusterAdmissionPolicies, err := f.listClusterAdmissionPolicies(ctx)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("failed to retrieve cluster-wide ClusterAdmissionPolicies: %w", err)
 	}
 	for _, policy := range clusterAdmissionPolicies {
 		policies = append(policies, &policy)
 	}
 
+	clusterAdmissionPolicyGroups, err := f.listClusterAdmissionPolicyGroups(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("failed to retrieve cluster-wide ClusterAdmissionPolicyGroups: %w", err)
+	}
+	for _, policy := range clusterAdmissionPolicyGroups {
+		policies = append(policies, &policy)
+	}
+
 	return f.groupPoliciesByGVR(ctx, policies, false)
 }
 
 // findNamespacesForAllClusterAdmissionPolicies returns all the ClusterAdmissionPolicies that evaluate resources in the given namespace.
```

Comment:
nit, you have to change the name of the function also inside of the inline docs. 

Odd, the linter should have spotted it, isn't it?